content/blog/2021/09/2021-09-20-election-period-opened.adoc

```diff
@@ -0,0 +1,95 @@
+---
+:layout: post
+:title: "Jenkins Election 2021"
+:tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- olblak
+description: >
+  2021 Jenkins Board and Officer elections. Nominations and voter registration are open!
+links:
+  discourse: true
+---
+
+image:/images/logos/needs-you/Jenkins_Needs_You-transparent.png[role="center", height=300, float="right", align="center"]
+
+Dear all,
+
+Time flies and the Jenkins elections period is there.
+We already had two successful editions in a row. I want us to continue on that path.
+This is a tremendous opportunity for community members to influence the direction of the project for the next two years.
+To make this year’s election even better, we slightly modified the process by leveraging our new community platform aka link:https://community.jenkins.io[community.jenkins.io].
+
```

Comment:
Add a paragraph explaining whose seats will be up for reelection? It is @uhafner , me, and all officers

Reply:
Done on this [commit](https://github.com/jenkins-infra/jenkins.io/pull/4559/commits/0f2335315fb7fd6aefa17db02fc1f696a57fff74)